packages/components/src/components/form/_form.scss

```diff
@@ -24,7 +24,7 @@
     @include type-style('body-short-01');
     display: flex;
     flex-direction: column;
-    flex: 1;
+    flex: 1 1 auto;
```

Comment:
```suggestion
    // We specify `auto` as the default value so that the form item does
    // not collapse in IE11 due to a `flex-basis` of 0 only working with
    // `flex-direction: row`
    flex: 1 1 auto;
```